src/argus_htmx/templates/htmx/incidents/_incidents_update_menu.html

```diff
@@ -1,7 +1,8 @@
 <ul class="menu menu-horizontal gap-2">
     {% block update_menu_content %}
-        <li><button class="btn btn-accent">Add Ticket</button></li>
-        <li><button class="btn btn-accent">Acknowledge</button></li>
-        <li><button class="btn btn-accent">Close/re-open</button></li>
+        <li>{% include 'htmx/incidents/_incident_ticket_edit_modal.html' with dialog_id="add-ticket-dialog" button_class="btn-accent" button_title="Change tickets" header="Change tickets" explanation="Write an URL of an existing ticket, or nothing to remove existing ticket urls" cancel_text="Cancel" submit_text="Submit" %}</li>
```

Comment:
`button_class` does not seem implemented in `_base_form_modal.html`, `btn-accent` does not seem to be applied

Reply:
What do you mean by `button_class`?

Reply:
`with dialog_id="add-ticket-dialog" button_class="btn-accent"`

Reply:
I could make the class (currently: "btn-primary") on the visible buttons overridable. The base modal is the same as is used in the details modals.

Reply:
Ok, maybe it's not clear what I mean. In the current main branch, the bulk buttons have `class="btn btn-accent"`. In this PR the bulk buttons only have `class="btn"` I thought that by passing a `button_class="btn-accent"` to the `include 'htmx/incidents/_incident_ticket_edit_modal.html` statement the idea was to place `btn-accent` on the bulk button. However, for this to work, inside  `_base_form_modal.html` the `button_class` context variable needs to be read, something like 

```python
<button class="btn {{ button_class }}"
```

Reply:
Solved with new commit on #139.